features/steps/step_selenium_alias_hipchat.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: UTF-8 -*-
+
+# @mark.steps
+# ----------------------------------------------------------------------------
+# STEPS:
+# ----------------------------------------------------------------------------
+from behave import given, when, then
+
+
+
+@given('we are on login page')
+def step_impl(context):
+    context.login_page.navigate()
```

Comment:
Do we have same code elsewhere? https://en.wikipedia.org/wiki/Don%27t_repeat_yourself